### ISSUE TITLE
Install error (apt) on Ubuntu 22.04/arm64/Firefox 119.deb (not ff snap, but there the same error)

### ISSUE THREAD

**Issue by bagong** [filips123/PWAsForFirefox#414]
### Description

Trying to install in ubuntu 22.04 on a arm64 system (orange pi 5) and following the instructions litterally, I get the following error from dpkg:

```
xyz@abcde:~$ sudo apt install firefoxpwa
[sudo] password for xyz: 
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following NEW packages will be installed:
  firefoxpwa
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/7.346 kB of archives.
After this operation, 23,2 MB of additional disk space will be used.
(Reading database ... 378135 files and directories currently installed.)
Preparing to unpack .../firefoxpwa_2.8.0_arm64.deb ...
Unpacking firefoxpwa (2.8.0) ...
dpkg: error processing archive /var/cache/apt/archives/firefoxpwa_2.8.0_arm64.deb (--unpack):
 unable to open '/usr/lib64/mozilla/native-messaging-hosts/firefoxpwa.json.dpkg-new': No such file or directory
No apport report written because the error message indicates an issue on the local system
         Errors were encountered while processing:
 /var/cache/apt/archives/firefoxpwa_2.8.0_arm64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

### Steps to Reproduce

Follow the install instructions for apt based systems literally until "sudo apt install firefoxpwa"

### Affected Websites

* ...
* ...

### Environment

* Operating system: Ubuntu 22.04
* System architecture: arm64
* Desktop environment:  Gnome 42.9
* Installation method:  apt in terminal
* PWAsForFirefox extension version: 2.8
* PWAsForFirefox native version: 2.8
* PWAsForFirefox runtime version: dunno
* Firefox version: 119.0

**Comment by filips123** [comment]
Is ˋ/usr/lib64ˋ a symlink to ˋ/usr/libˋ (or somewhere else) on your system? Appearently, this error can happen in such cases, as the DEB package will try to install a file both to ˋ/usr/lib/mozilla/native-messaging-hosts/firefoxpwajsonˋ and ˋ/usr/lib64/mozilla/native-messaging-hosts/firefoxpwa.jsonˋ.

**Comment by bagong** [comment]
Correct:

````
xyz@abcde:~$ ll /usr/lib64
lrwxrwxrwx 1 root root 3 Okt 15 20:16 /usr/lib64 -> lib/
````
Yet /usr/lib doesn't contain firefoxpwa.json either:
````
xyz@abcde:~$ ls /usr/lib/mozilla/native-messaging-hosts/
org.gnome.chrome_gnome_shell.json
````
 I don't understand the error, I wouldn't expect a firefoxpwa file to reside in such a location during install of the very app - but as I said, no idea what it's about. Is there something I can do? This does look like a bug, right?

**Comment by filips123** [comment]
Basically, PWAsForFirefox contains a manifest in both lib and lib64 directories (as this is needed appearently as Firefox might scan only lib64 in some cases). When dpkg tries to install a file, it first copies it to the correct directory and adds ˋ.dpkg-newˋ suffix. However, as lib and lib64 are the same directory, one of the packaged files would overwrite the other one, so dpkg fails and doesn't install anything.

It's kinda partially a bug because the package doesn't expect systems to have this symlink. I'll try to fix this somehow so the install will work properly.

Until then, you can try to manually exit the DEB archive to remove the lib64 version of the manifest, or manually install the native part with the instructions in the README.

**Comment by filips123** [comment]
This should be fixed in 2.9.0.

**Comment by bagong** [comment]
Thanks. The error is gone!
